python/fledge/services/core/api/proxy.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+
+# FLEDGE_BEGIN
+# See: http://fledge-iot.readthedocs.io/
+# FLEDGE_END
+import json
+import logging
+import aiohttp
+
+from aiohttp import web
+from fledge.common import logger
+from fledge.services.core import server
+from fledge.services.core.service_registry import exceptions as service_registry_exceptions
+from fledge.services.core.service_registry.service_registry import ServiceRegistry
+
+
+__author__ = "Ashish Jabble"
+__copyright__ = "Copyright (c) 2022 Dianomic Systems Inc."
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+_logger = logger.setup(__name__, level=logging.INFO)
+SVC_TYPE = "BucketStorage"
```

Comment:
We should not have anything that mentions the service type as part of this is to allow new services in FogLAMP that Fledge does not know about